liota/dccs/iotcc.py

```diff
@@ -410,16 +413,23 @@ def publish_unit(self, reg_entity_obj, metric_name, unit):
         :return:
         """
         str_prefix, str_unit_name = parse_unit(unit)
-        if not isinstance(str_prefix, basestring):
-            str_prefix = ""
-        if not isinstance(str_unit_name, basestring):
-            str_unit_name = ""
-        properties_added = {
-            metric_name + "_unit": str_unit_name,
-            metric_name + "_prefix": str_prefix
-        }
-        self.set_properties(reg_entity_obj, properties_added)
-        log.info("Published metric unit with prefix to IoTCC")
+        if not isinstance(str_prefix, basestring) and isinstance(str_unit_name, basestring):
```

Comment:
So the mentioned issue can happen only when SI units which do not have a prefix are used? No Error will happen if there is no SI unit is set, or if the SI unit is having a prefix. Do we have a list of units where the issue can happen and where it cannot?